app/src/main/res/layout/appcoins_radio_button.xml

```diff
@@ -30,13 +30,112 @@
       style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="16dp"
-      android:layout_marginBottom="4dp"
+      android:layout_marginStart="8dp"
+      android:layout_marginBottom="8dp"
       android:text="@string/appc_token_name"
       android:textSize="16sp"
-      app:layout_constraintBottom_toTopOf="@id/message"
+      app:layout_constraintBottom_toTopOf="@id/appc_fiat_balance"
       app:layout_constraintStart_toEndOf="@id/icon"
       />
+  <androidx.constraintlayout.widget.Group
+      android:id="@+id/appc_balances_group"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      app:constraint_referenced_ids="appc_fiat_balance,appc_text_separator,appc_balance,balance_eth,eth_fiat_balance,eth_text_separator,eth_balance"
+      />
+  <TextView
+      android:id="@+id/appc_fiat_balance"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:fontFamily="sans-serif-medium"
+      android:lineSpacingExtra="4sp"
+      app:layout_constraintBottom_toTopOf="@id/balance_eth"
+      app:layout_constraintStart_toStartOf="@id/title"
+      tools:text="31.90 EUR"
+      />
+  <TextView
+      android:id="@+id/appc_text_separator"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="|"
```

Comment:
This attribute should be defined as a untranslatable string in the resources folder